packages/sdk-browser/src/sdk-client.ts

```diff
@@ -183,6 +195,64 @@ export class TurnkeyBrowserClient extends TurnkeySDKClientBase {
     await setStorageValue(StorageKeys.CurrentUser, currentUser);
     return readOnlySessionResult!;
   };
+
+  // createEmbeddedAPIKey creates an embedded API key encrypted to a target key (typically embedded within an iframe).
+  // This returns a bundle that can be decrypted by that target key, as well as the public key of the newly created API key.
+  createEmbeddedAPIKey = async (
```

Comment:
if the `createEmbeddedAPIKey` function is general and doesn't require specific knowledge of the current turnkey instance I think it would be better for it to live in `src/utils.ts` and then have users interact with it via `passkeyClient.createPasskeySession()` or is there a reason why the specific crypto logic needs to be exposed at the top-level of the `turnkey` instance?

Reply:
I wanted to abstract it out + still expose it because in theory, a user might want to be able to create an API key within the browser and then include it within a new suborg creation. From the freshly updated PR description:

> This does not encapsulate logic to, say, create a new suborg and have a fresh passkey session for them out of the gate. However, the abstraction is there: you could call createEmbeddedAPIKey, grab the resulting public key and include it as a user's API key within a CreateSubOrganization call.